source/Ship.h

```diff
@@ -58,6 +58,15 @@ class Visual;
 // limits of what the AI knows how to command them to do.
 class Ship : public Body, public std::enable_shared_from_this<Ship> {
 public:
+	enum class HyperStage : int {
+		NotJumping = 0,
```

Comment:
I feel like if one is defined, the rest should also be?

Reply:
We just need 0/1 state so I don't think its needed
look at preferences.h